taichi/util/offline_cache.cpp

```diff
@@ -5,6 +5,19 @@ namespace taichi::lang::offline_cache {
 constexpr std::size_t offline_cache_key_length = 65;
 constexpr std::size_t min_mangled_name_length = offline_cache_key_length + 2;
 
+void disable_offline_cache_if_needed(CompileConfig *config) {
+  TI_ASSERT(config);
+  if (config->offline_cache) {
+    if (config->print_preprocessed_ir || config->print_ir ||
+        config->print_accessor_ir) {
+      config->offline_cache = false;
+      TI_WARN(
+          "Disable offline_cache because print_preprocessed_ir or print_ir or "
+          "print_accessor_ir is enabled");
```

Comment:
```suggestion
          "print_accessor_ir or print_evaluator_ir is enabled");
```

Reply:
Thanks. But the evaluators won't be cached, so I think `print_evaluator_ir` will always work correctly. WDYT?

(see:
1. LLVM

https://github.com/taichi-dev/taichi/blob/master/taichi/codegen/codegen.cpp#L103-L104

3. GFX

https://github.com/taichi-dev/taichi/blob/master/taichi/cache/gfx/cache_manager.cpp#L151-L155

5. Metal (in #6227)
https://github.com/taichi-dev/taichi/blob/e559866e933b13d507bf8308d3da3c88e42fc03a/taichi/cache/metal/cache_manager.cpp#L28-L30

Reply:
Thanks for pointing this out. I agree that `print_evaluator_ir` will work properly.